docs/src/details/integration/chip/renesas/supported_boards.rst

```diff
@@ -0,0 +1,79 @@
+================
+Supported Boards
+================
+
+.. list-table::
+   :widths: 10 30 30 30 30 30
+
+   * - Board Name
+     - CPU
+     - Memory
+     - Display
+     - `Board <https://lvgl.io/boards>`__ video
+     - Links
+   * - **EK-RA8D1**
+     - 480MHz, Arm Cortex-M85 core
+     -
+         | 1MB internal, 64MB external SDRAM
+         | 2MB internal, 64MB External Octo-SPI Flash
+     -
+         | 4.5”
+         | 480x854
+         | 2-lane MIPI
+     - .. raw:: html
+
+           <iframe width="320" height="180" src="https://www.youtube.com/embed/WkJPB8wto_U" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+
+     - `Demo repository for EK-RA8D1 <https://github.com/lvgl/lv_port_renesas_ek-ra8d1>`__
+   * - **EK-RA6M3G**
+     - 120MHz, Arm Cortex-M4 core
+     -
+         | 640kB internal SRAM
+         | 2MB internal, 32MB external QSPI Flash
+     -
+         | 4.3”
+         | 480x272
+         | Parallel RGB565
+     - .. raw:: html
+
+           <iframe width="320" height="180" src="https://www.youtube.com/embed/0kar4Ee3Qic" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+
+     - `Demo repository for EK-RA6M3G <https://github.com/lvgl/lv_port_renesas_ek-ra6m3g>`__
+   * - **RX72N Envision Kit**
+     - 240MHz, Renesas RXv3 core
+     -
+         | 1MB internal SRAM
+         | 4MB internal, 32MB external QSPI Flash
+     -
+         | 4.3”
+         | 480x272
+         | Parallel RGB565
+     - .. raw:: html
+
+           <iframe width="320" height="180" src="https://www.youtube.com/embed/__56v8DsfH0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+
+     - `Demo repository for RX72N Envision Kit <https://github.com/lvgl/lv_port_renesas_rx72n-envision-kit>`__
+   * - **RZ/G2L-EVKIT**
+     - 1.2 GHz Arm Cortex-A55
```

Comment:
Since it is not rare to find multi-core SMP microprocessors out there, it would not be good also to mention how many cores the SoC has?